deploy/helm/sumologic/templates/setup/job.yaml

```diff
@@ -14,6 +14,7 @@ metadata:
 {{ include "sumologic.annotations.app.setup.helmsh" "3" | indent 4 }}
   labels:
     app: {{ template "sumologic.labels.app.setup.job" . }}
+    sumologic.com/app: setup
```

Comment:
I would creat new helm template to have both of the labels

Reply:
done